src/lib/viewers/ResultsViewer.ts

```diff
@@ -193,11 +193,6 @@ function sortViolations(violations: Violation[]): Violation[] {
 	});
 }
 
-// TODO: We should update core module to have this function directly on the Violation object and then remove this helper
```

Comment:
With the other enhancements that I just pulled in, I was able to accomplish this TODO item as well.